src/apps/leaderboards/models.py

```diff
@@ -5,19 +5,23 @@
 class Leaderboard(models.Model):
     ADD = "Add"
     ADD_DELETE = "Add_And_Delete"
+    ADD_DELETE_MULTIPLE = "Add_And_Delete_Multiple"
     ADD_BY_REVIEW = "Add_By_Review"
     FORCE_LAST = "Force_Last"
     FORCE_LATEST_MULTIPLE = "Force_Latest_Multiple"
     FORCE_BEST = "Force_Best"
     SUBMISSION_RULES = (
         (ADD, "Only allow adding one submission"),
         (ADD_DELETE, "Allow users to add a single submission and remove that submission"),
+        (ADD_DELETE_MULTIPLE, "Allow users to add a multiple submissions and remove those submission"),
         (ADD_BY_REVIEW, "Only allow adding one submission that is approved by review"),
         (FORCE_LAST, "Force only the last submission"),
         (FORCE_LATEST_MULTIPLE, "Force latest submission to be added to leaderboard (multiple)"),
         (FORCE_BEST, 'Force only the best submission to the leaderboard')
     )
 
+    AUTO_SUBMISSION_RULES = [FORCE_LAST, FORCE_BEST, FORCE_LATEST_MULTIPLE]
```

Comment:
Is `FORCED_SUBMISSION_RULES` a better name for this constant? Why did you call it `AUTO_SUBMISSION_RULES`?

Reply:
I called it auto because these rules are handled automatically by the leaderboard. The user can't manually remove or add a submission.